app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <integer name="fade_in_duration">1000</integer>
```

Comment:
Please include the time unit in the name, e.g. ``fade_in_duration_ms`` since this appears to be in milliseconds.

Reply:
Okay